lumicks/pylake/kymotracker/detail/calibrated_images.py

```diff
@@ -59,6 +60,9 @@ def get_rect(self, rect):
         t0_pixels, t1_pixels = self.from_seconds(t0), self.from_seconds(t1)
         p0_pixels, p1_pixels = self.from_coord(p0), self.from_coord(p1)
 
+        if np.any(np.array([t0_pixels, t1_pixels, p0_pixels, p1_pixels]) < 0):
+            raise IndexError(f"Selection needs to be in bounds of the image")
+
```

Comment:
I think the error message should be more specific something like `"negative indices are not supported."`. This would make it clear that we purposefully do not support negative indices, unlike lists or numpy arrays

Reply:
I wouldn't talk about indices, since we're selecting by coordinate here, not index.

I'll add the phrase `indexing by negative coordinates not supported`.